src/test/java/gil/server/RootEndpointControllerTest.java

```diff
@@ -0,0 +1,23 @@
+package gil.server;
+
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+public class RootEndpointControllerTest {
+    RootEndpointController rootEndpointController = new RootEndpointController();
+
+    @Test
+    public void getShouldResultInResponseWith404Status() {
```

Comment:
This test is incorrect as your actual assertion tests that a "200" status code is returned.